azure-specialized-workloads/avd/recommendations.yaml

```diff
@@ -1,154 +1,181 @@
-- description: Use Private link when connecting to File Share or Key Vault
-  aprlGuid: dc55be60-6f8c-461e-a9d5-a3c7686ed94e
+- description: Monitor service health and resource health for AVD
```

Comment:
Confirmed with @danycontre that he intends these to be different recommendations which is why they have new guid